TicTacToe2/TicTacToeGame.cs

```diff
@@ -6,14 +6,18 @@ namespace TicTacToe2
 {
     class TicTacToeGame
     {
-        private readonly string[] space = new string[9] { "1", "2", "3", "4", "5", "6", "7", "8", "9" };
+        private readonly string[] space = new string[16] { "1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14", "15", "16" };
         private string player = "X";
         private int turnCount = 0;
+        private int gridNumber;
+        private int choice = 0;
         private bool winner = false;
         private bool draw = false;
+        private bool correctNumber = false;
 
         public void Play()
         {
+            ChooseBoard();
```

Comment:
The Play() function looks good. Only adding the ChooseBoard() function keeps things simple.